dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -755,3 +755,67 @@ CWWKD1077.defaultds.not.found.explanation=Jakarta Data repositories default to \
  configuration.
 CWWKD1077.defaultds.not.found.useraction=Configure the Jakarta EE default \
  data source or specify a different data store for the repository.
+
+CWWKD1078.datastore.not.found=CWWKD1078E: The {0} application artifact has a \
+ {1} repository interface with a {2} data store that does not match the Id of a \
```

Comment:
Change
the Id of
to
the ID of

(IBM style is to use both uppercase letters "ID" when used as an abbreviation of identifier or identification)